app-leechcraft/lc-azoth/lc-azoth-9999.ebuild

```diff
@@ -43,49 +60,52 @@ RDEPEND="${COMMON_DEPEND}
 	)
 	crypt? ( app-crypt/qca:2[gpg] )
 	latex? (
-		virtual/imagemagick-tools
+		|| (
+			media-gfx/imagemagick
+			media-gfx/graphicsmagick[imagemagick]
+		)
```

Comment:
why did you take away the virtual? It's either virtual or USE flags, but no naked `|| ( )` deps anymore